src/Bridge/Symfony/Messenger/DataPersister.php

```diff
@@ -88,6 +88,6 @@ public function persist($data, array $context = [])
      */
     public function remove($data, array $context = [])
     {
-        $this->messageBus->dispatch(new Envelope($data, new RemoveStamp()));
+        $this->messageBus->dispatch(new Envelope($data, [new RemoveStamp()]));
```

Comment:
```php
$this->messageBus->dispatch(
    (new Envelope($data))
        ->with(new RemoveStamp())
);
```

Reply:
I'm not sure why it's still possible to pass in the stamps through the constructor...